Doc/library/winreg.rst

```diff
@@ -165,6 +168,9 @@ This module offers the following functions:
 
    *This method can not delete keys with subkeys.*
 
+   The key identified by the *key* parameter must have been opened with
+   :const:`KEY_SET_VALUE` access.
+
```

Comment:
Per the message I left on the tracker, this should state that `key` can have any access. It's only used to resolve the relative path in `sub_key`. 

Also, using `KEY_WOW64_64KEY` instead of 0 as the default value of `access` is wrong (i.e. it's the wrong default behavior for 32-bit Python). And the usage of `KEY_WOW64_64KEY` and `KEY_WOW64_32KEY` -- the only relevant access modes for `sub_key` -- should probably be discussed in terms how the relative path in `sub_key` is traversed starting from the path of the key that's referenced by the `key` handle.